weed/server/filer_server_handlers_read.go

```diff
@@ -119,7 +119,16 @@ func (fs *FilerServer) GetOrHeadHandler(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	if r.URL.Query().Has("metadata") {
+	query := r.URL.Query()
+	if query.Get("metadata") == "true" || query.Has("metadata") {
+		if query.Get("resolveManifest") == "true" || query.Has("resolveManifest") {
```

Comment:
these query.Has() logic is wrong. If `metadata=false`, this would still work.

Please test the code before submitting PR.

Reply:
😂sorry